pushy/src/main/java/com/eatthepath/pushy/apns/util/ApnsPayloadBuilder.java

```diff
@@ -115,6 +118,26 @@
      */
     public static final String DEFAULT_SOUND_FILENAME = "default";
 
+    /**
+     * The passive interruption level.
+     */
+    public static final String INTERRUPTION_LEVEL_PASSIVE = "passive";
```

Comment:
I see where you're going, but I think it might make sense to turn this into `enum InterruptionLevel` instead.

Reply:
Of course!